doc/api/tls.md

```diff
@@ -932,10 +932,14 @@ changes:
 -->
 
 * `options` {Object}
-  * `pfx` {string|Buffer} Optional PFX or PKCS12 encoded private key and
-    certificate chain. `pfx` is an alternative to providing `key` and `cert`
+  * `pfx` {Buffer|Buffer[]|Object[]} Optional PFX or PKCS12 encoded private key
```

Comment:
How come `string` is not supported anymore?

Reply:
PKCS12 is very rarely stored in PEM format, by default it's DER-encoded. But I forget about binary strings in JS. Will revert it, ACK.